### PR TITLE
fauna-shell 0.3.0 (new formula)

### PR DESCRIPTION
**Pull request by alvarofauna**
Adds a formula for [fauna-shell](https://www.npmjs.com/package/fauna-shell), a tool that let users operate and interact from the CLI with [FaunaDB](https://fauna.com/)

### CODE REVIEW

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/fauna-shell.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"fauna --help"
```
[the remark this answers is not in the document]
Comment (alvarofauna) [reply Formula/fauna-shell.rb]: this should run a test suit or make sure that the output of a command is correct?

[the remark this answers is not in the document]
Comment (fxcoudert) [reply Formula/fauna-shell.rb]: The later.

**Comment by alvarofauna** [comment]
@fxcoudert I added tests as requested

**Comment by alvarofauna** [comment]
Fixed the tests since there were unused variables

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @alvarofauna for the pull request, and congratulations on your first contribution to Homebrew! It is now merged

**Comment by alvarofauna** [comment]
Thanks @fxcoudert 

If I'd want to push an updated version of the formula, is that just a new PR? Basically every time we release a new version of the npm package.

**Comment by fxcoudert** [comment]
@alvarofauna yes, a new PR with each new version

**Pull request closed without merging**